modules/kubernetes-addons/helm_addon/locals.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  service_account_name = "${var.name}-sa"
+  namespace            = "${var.name}-ns"
+
+  default_helm_config = {
+    name                       = var.name
+    chart                      = var.name
+    namespace                  = local.namespace
+    timeout                    = "1200"
+    create_namespace           = true
+    description                = "Kubernetes Helm AddOn"
+    lint                       = false
+    wait                       = true
+    wait_for_jobs              = false
+    verify                     = false
+    keyring                    = ""
+    repository_key_file        = ""
+    repository_cert_file       = ""
+    repository_ca_file         = ""
+    repository_username        = ""
+    repository_password        = ""
+    disable_webhooks           = false
+    reuse_values               = false
+    reset_values               = false
+    force_update               = false
+    recreate_pods              = false
+    cleanup_on_fail            = false
+    max_history                = 0
+    atomic                     = false
+    skip_crds                  = false
+    render_subchart_notes      = true
+    disable_openapi_validation = false
+    dependency_update          = false
+    replace                    = false
+    postrender                 = ""
```

Comment:
We could only keep these lines `locals.tf ` at `modules/kubernetes-addons/helm_addon/locals.tf`
```
e.g., default_helm_config ={
    -- above mentioned lines
}
```

- Send this local variable config to each add-on module. 

- Merge this root level default config with locals default values for each add-on

This approach actually removes defining too many variable on each add-on at locals.tf and offload some to the root level